src/map-feature.js

```diff
@@ -285,26 +285,30 @@ export class MapFeature extends HTMLElement {
   // native cs: used by FeatureLayer._geometryToLayer(...),
   //            the fallback cs for map-geometry if its cs attribute is not specified
   _getFallbackCS() {
-    let csMeta;
+    let csMeta, cs;
     if (this._parentEl.nodeName === 'MAP-LINK') {
       // feature attaches to link's shadow
-      csMeta =
-        this._parentEl.shadowRoot.querySelector('map-meta[name=cs]') ||
-        this._parentEl.parentElement.getMeta('cs');
```

Comment:
re: `this._parentEl.parentElement.getMeta('cs');` this was not doing anything, because the _parentEl.parentElement (\<map-extent>) getMeta function only looks for meta elements of namee=`extent` or `zoom`.  We perhaps should look at extending that to include `cs` in this case.

Reply:
Have changed \<map-extent>.getMeta to return name=cs.  tbd: if it causes problems elsewhere